spec.bs

```diff
@@ -1106,7 +1115,7 @@ and a <dfn for=InterestGroupBiddingScriptRunnerGlobalScope>priority signals</dfn
 </div>
 
 <div algorithm>
-  To <dfn>evaluate a script</dfn> with a global type |globalType|, [=string=] |script|, [=string=]
+  To <dfn>evaluate a script</dfn> with a global object |global|, [=string=] |script|, [=string=]

```

Comment:
Can you make "global object" link to this type: https://html.spec.whatwg.org/multipage/webappapis.html#concept-realm-global